DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -34,19 +34,24 @@ public static string GenerateSizeDisplayString(long byteCount)
             // Do not include decimal place below GB
             var decimalPlaces = place <= 2 ? 0 : 1;
             var number = Math.Round(byteCount / Math.Pow(1024, place), decimalPlaces);
-            return (Math.Sign(byteCount) * number) + Units[place];
+            return Math.Sign(byteCount) * number + Units[place];
         }
 
         public static string? ConvertNumberToMonthsString(int numberOfMonths)
         {
             return numberOfMonths == 0 ? null : $"{numberOfMonths} month{GetPluralitySuffix(numberOfMonths)}";
         }
 
-        public static string GetDelegateNameString(string? firstName, string lastName)
+        public static string GetFullNameForDisplay(string? firstName, string lastName)

```

Comment:
Hmm, these names make it seem like we're always displaying it as `Firstname Lastname` but when we sort we're going to sort by Lastname first (but still display `Firstname Lastname`.
Can we make it more clear that the other method gives the full name to display for items which can be sorted by name, and that this method gives the full name to display for items that aren't being sorted by name?

Reply:
I've renamed them to hopefully make them clearer. If you think they still aren't obvious enough, I might need some suggestions, as I've wracked my brain for direct and clean names and these are the best I can think of.